lib/osf-components/app/components/brand-color-inputs/template.ts

```diff
@@ -0,0 +1 @@
+export { default } from 'osf-components/components/brand-color-inputs/template';
```

Comment:
The extension here should .js `lib/osf-components/app/components/brand-color-inputs/template.js`

Reply:
Ahh, the extension should be `.js` 👀  It seems like we are re-exporting `component.js` and `template.js` for our glimmer components, and I tried removing the template export, and the component broke. It seems it might be needed, but not exactly sure why?